newton/_src/viewer/gl/gui.py

```
@@ -55,6 +56,18 @@ def __init__(self, window):
 
         self.io = self.imgui.get_io()
 
+        # Fix inverted scroll direction in the pyglet imgui backend before
+        # attaching callbacks so pyglet captures the corrected handler.
+        # The replacement must be named "on_mouse_scroll" because pyglet
+        # matches handlers by __name__.
+        io = self.io
+
+        def on_mouse_scroll(x, y, scroll_x, scroll_y):
```

Ruff (0.15.2)

[warning] Unused function argument: `x`

(ARG001)